query06.sql

```diff
@@ -1,6 +1,20 @@
+-- Johnathan Clementi --
+-- MUSA 509 Geospatial Cloud Computing & Vizualization --
+-- HW #1 2021-09-19 --
+-- Query #6 --
+
 /*
   How many trips were shorter than 10 minutes?
 */
 
 -- Enter your SQL query here
-select ...
+with indego_trips_Q2_19and20 as (
+  select duration from indego_trips_2019_q2
+  union
+  select duration from indego_trips_2020_q2
```

Comment:
Ah, you've fallen into a gotcha of the `union` keyword. According to the [docs](https://www.postgresql.org/docs/current/queries-union.html):

> UNION effectively appends the result of query2 to the result of query1 (although there is no guarantee that this is the order in which the rows are actually returned). Furthermore, **it eliminates duplicate rows from its result, in the same way as DISTINCT, unless UNION ALL is used.**

You're only selecting `duration`, so this query will only select the _unique_ durations. So, if seven trips have a duration of 20 minutes, they'll only be counted once. This is one place where `union all` would be important.